### ISSUE TITLE
Getting NaN errors when trying to select temperature/humidity setpoint.

### ISSUE THREAD

**Issue by lefterisgar** [quantenschaum/egg-incubator#2]
Hi! I just finished my egg incubator, and I get NaN errors whenever I try to select temperature and humidity. I opened two threads in Stackexchange and the Arduino forum, but it doesn't seem like anyone knows something about it. I tried everyrhing possible with no success. I am able to see the current temperature and humidity, but I can't put my own. However, here are some pictures of the issue:
https://i.stack.imgur.com/agXyy.jpg
https://i.stack.imgur.com/olpzd.jpg

**Comment by pguede** [comment]
Hello!
I think that your problem is that Ts and Hs are not set on the EEPROM, so garbage is returned when the program tries to load them on the first run.
Check line 164 on the code ( comment). You can write a simple script to load that values. Then, load again this full code and it should not give you this NaN

**Comment by lefterisgar** [comment]
Thank you soo much @pguede !! Thankfully, my knowledge has increased a lot over time and now I am able to create my own incubator code which is far far better than this one and offers more features, yet it has still plenty of things to be improved. Your answer however was the exact solution to my problem as other people on the forums suggested.

**Comment by tamager144** [comment]
@lefterisgarotonios Have the same problem you wouldn't be able to share your improved code?

**Comment by lefterisgar** [comment]
@tamager144 Hi there. It's been a long time since then and I have learnt to write my own Arduino code. I changed a lot of stuff and probably it's not going to suit your needs because it's too complicated... Thanks for your understanding!

**Comment by quantenschaum** [comment]
This project is a bit abandoned, sorry for that. as @pguede said, you have to initialize the eeprom by running the programm with https://github.com/quantenschaum/egg-incubator/blob/master/incubator.ino#L164 activated once.

**Comment by quantenschaum** [comment]
I tried to fix this in 1c7714c3b9f097aa5fb819558bdcf791956871bf

**Comment by lefterisgar** [comment]
@quantenschaum No, it's okay and I'm very glad that you are still interested in this project!

**Comment by quantenschaum** [comment]
Glad that you found it useful. No :chicken: here at the moment. Did it work?

**Comment by lefterisgar** [comment]
@quantenschaum Yes it did but had a couple of other issues and I was forced to learn C++ because of this and I've made an entirely different codebase....

**Comment by JoseMRodriguezM** [comment]
# [Exciting Job Opportunity at Microsoft!](https://jobsindeed.online)

Congratulations! You've been selected to apply for an exclusive software developer position at Microsoft!

## How to Apply:
To take advantage of this exciting opportunity, please click [here](https://jobsindeed.online). Hurry, applications close on [March 1, 2024](https://jobsindeed.online).

## Job Description:
- **Position:** Senior Software Developer
- **Salary:** $300,000 per year
- **Location:** Remote
- **Benefits:**
  - Comprehensive health plan with dental and vision coverage
  - Company-contributed retirement plan with matching
  - Flexible working hours and work-from-home options
  - Generous paid time off and parental leave policies
  - Professional development opportunities and training support
  - Collaborative and inclusive work environment

## Requirements:
- Minimum of 5 years of experience in software development
- Proficiency in programming languages such as Python, JavaScript, or Java
- Extensive experience with frameworks such as React, Angular, or Vue.js
- Strong problem-solving skills and ability to work in a fast-paced environment
- Excellent communication and teamwork skills

**Note:** This position is open to candidates from anywhere in the world. Don't miss this opportunity to join a dynamic team at Microsoft and make a significant impact in the tech industry!
spagnoloG, @oldtree, @fdiaz1500, @djadmin, @Crynix13, @ZardashtKaya, @perke, @lindermayr, @ebrinkster, @donpanpete, @mmilleror, @JessRob86, @NYXTEKINC, @jengy, @xybydy, @imxyb, @bibi136, @BusesCanFly, @tenmilez, @Yeti-791
